Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["packages/*", "contracts/*", "tools/deploy"]
```

Comment:
Question (maybe resolved after reading the code):

How much of this is from the general climb cli? Or belongs there? Or does this extend it and reuse it?

Reply:
Imho there isn't really overlap, as of this PR it's very thin, pretty much just "rust CLI boilerplate" stuff. As we add more and more commands, my gut feeling is that the overlap will be with backend and tests, and we'll maybe decide to move things into the `api` crate or wherever you think shared layer-specific code should go